app/views/components/blockRegistration.html

```diff
@@ -1,8 +1,8 @@
 <div class="form-group">
-  <label class="control-label">{{block.title}} <em class="required" ng-if="block.required">*</em></label>
+  <label class="control-label"><span ng-bind-html="block.title"></span><em class="required" ng-if="block.required">*</em></label>
 
   <ng-switch on="block.type">
-    <p ng-switch-when="paragraphContent" ng-bind="block.content" class="display-linebreaks"></p>
+    <p ng-switch-when="paragraphContent" ng-bind-html="block.content" class="display-linebreaks"></p>
```

Comment:
@OzzieOrca Here as well the master (with merges) shows:

`<p ng-switch-when="paragraphContent" ng-bind="block.content.paragraph" class="display-linebreaks"></p>`

Which line is desired?